scripts/ci-apidiff.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+APIDIFF="hack/tools/bin/go-apidiff"
+
+cd "${REPO_ROOT}" && make ${APIDIFF}
+echo "*** Running go-apidiff ***"
+
+${APIDIFF} "${PULL_BASE_SHA}"
```

Comment:
Any particular reason for a separate script here over having a make target for running go-apidiff?

Reply:
This wouldn't work in local environments, so I opted to keep it in the script file and just have the Makefile call the binary directly

Reply:
makes sense, thanks for the clarification